src/briefcase/platforms/linux/__init__.py

```diff
@@ -1,5 +1,190 @@
-DEFAULT_OUTPUT_FORMAT = "appimage"
+import subprocess
+import sys
+from pathlib import Path
+from typing import List
+
+from briefcase.commands.create import _is_local_requirement
+from briefcase.commands.open import OpenCommand
+from briefcase.config import AppConfig
+from briefcase.exceptions import BriefcaseCommandError, ParseError
+
+DEFAULT_OUTPUT_FORMAT = "system"
+
+DEBIAN = "debian"
+RHEL = "rhel"
+ARCH = "arch"
+
+
+def parse_freedesktop_os_release(content):
+    """Parse the content of an /etc/os-release file.
+
+    :param content: The text content of the /etc/os-release file.
+    :returns: A dictionary of key-value pairse, in the same format returned by
+        `platform.freedsktop_os_release()`.
+    """
+    try:
+        return {
+            parts[0].strip(): parts[1].strip().strip('"').strip()
+            for parts in [
+                line.strip().split("=", 1) for line in content.split("\n") if line
+            ]
+        }
+    except (AttributeError, IndexError) as e:
+        raise ParseError(
+            f"Failed to parse output of FreeDesktop os-release file: {e}"
+        ) from e
 
 
 class LinuxMixin:
     platform = "linux"
+
+    def vendor_details(self, freedesktop_info):
+        """Normalize the identify of the target Linux vendor, version, and base.
+
+        :param freedesktop_info: The parsed content of the FreeDesktop
+            /etc/os-release file. This is the same format returned by
+            `platform.freedsktop_os_release()`.
+        :returns: A tuple of (vendor, version, vendor_base).
+        """
+        vendor = freedesktop_info["ID"]
+        try:
+            codename = freedesktop_info["VERSION_CODENAME"]
+            if not codename:
+                # Fedora *has* a VERSION_CODENAME key, but it is empty.
+                # Treat it as missing.
+                raise KeyError("VERSION_CODENAME")
+        except KeyError:
+            try:
+                # Arch uses a specific constant in VERSION_ID
+                if freedesktop_info["VERSION_ID"] == "TEMPLATE_VERSION_ID":
+                    codename = "rolling"
+                else:
+                    codename = freedesktop_info["VERSION_ID"].split(".")[0]
+            except KeyError:
+                # Manjaro doesn't have a VERSION_ID key
+                codename = "rolling"
+
+        # Process the vendor_base from the vendor.
+        id_like = freedesktop_info.get("ID_LIKE", "").split()
+        if vendor == DEBIAN or DEBIAN in id_like or "ubuntu" in id_like:
+            vendor_base = DEBIAN
+        elif vendor == RHEL or vendor == "fedora" or RHEL in id_like:
+            vendor_base = RHEL
+        elif vendor == ARCH or ARCH in id_like:
+            vendor_base = ARCH
+        else:
+            vendor_base = None
```

Comment:
I'm not sure we can - there's some odd descriptions in ID_LIKE - Linux mint is a "Ubuntu" like; RHEL is like "Fedora". Ultimately, we don't care what OS it's "like" - we want to know "Do I have to use rpm or deb?"

Reply:
Also - FWIW, this currently *doesn't* identify SuSE, because it is "ID_LIKE"... "suse". So you'll be able to get as far as running the app locally, but not package. 

I'm happy to put SuSE support in the "future work" bucket - we have to draw the line somewhere, or we'll be chasing distros until the end of time. Once this base feature exists, adding a new vendor base of "suse" should be reasonably straightforward for a motivated contributor.

Reply:
Sounds good to me 👍🏼 Mostly just wanted to at least mention SUSE since RPM support was added.